<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestClass</class>
 <widget class="QMainWindow" name="QtTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>610</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QtTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="digitCount">
     <number>15</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>122</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x²</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>122</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>300</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">modern {
    display: inline-block;
    margin: 10px;
    padding: 8px 15px;
    background: #B8ED01;
    border: 1px solid rgba(0,0,0,0.15);
    border-radius: 4px;
    transition: all 0.3s ease-out;
    box-shadow: 
        inset 0 1px 0 rgba(255,255,255,0.5), 
        0 2px 2px rgba(0,0,0,0.3), 
        0 0 4px 1px rgba(0,0,0,0.2); 
    /* Font styles */
    text-decoration: none;
    text-shadow: 0 1px rgba(255,255,255,0.7);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>412</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>412</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delenie">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>122</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_umnojenie">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>181</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>358</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_total">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>358</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>358</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>299</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>299</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>299</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>240</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>240</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>240</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>181</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>181</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>181</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_LCD_color">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_Background_Color">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_percent">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>51</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arroy">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>122</y>
      <width>77</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtTest.qrc"/>
 </resources>
 <connections/>
</ui>
